src/Tribe/Migration/Queue_4_12.php

```diff
@@ -0,0 +1,105 @@
+<?php
+/**
+ * Queue to handle Attendees Block REST migration.
+ *
+ * @since TBD
+ */
+
+namespace Tribe\Tickets\Migration;
+
+use Tribe\Tickets\Events\Attendees_List;
+use Tribe__Tickets__Main;
+use WP_Query;
+
+class Queue_4_12 extends Queue {
+
```

Comment:
:no_entry_sign: Missing doc comment for class Queue_4_12

<sub>hash: bbe05c7caf185a7a396eb2ca328b8a71</sub>